decode.rb

```diff
@@ -0,0 +1,57 @@
+MORSE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(char)
+  if MORSE.key? char
+    MORSE[char]
+  else
+    'Input is not morse code'
+  end
+end
+
+def decode_word(word)
+  word_arr = []
+  word.split.each do |n|
+    word_arr.push(MORSE[n])
+  end
+  word_arr.join
+end
+
+def decode_message(msg)
+  morse_msg = ''
+  msg.strip.tr('/', ' ').split('   ').each do |n|
+    morse_msg += "#{decode_word(n)} "
+  end
+  word
```

Comment:
When trying to run your application I am having the following error: `undefined local variable or method `word'`
Please make the necessary changes to your code in order to correct this error. 

Check the image below for a better understanding:
![Screenshot from 2022-01-20 20-08-29](https://user-images.githubusercontent.com/75709904/150447825-ecde6ebc-b218-4659-8df0-b5fb279ece00.png)


**hint:** You are returning the wrong variable name `word`, the only variable you have inside your `decode_message` method is `hint: You are using`.